admin-ui.php

```diff
@@ -131,21 +145,21 @@ function admin_page() {
 			} else {
 				Keyring::message( __( 'This service does not currently support connection testing.', 'keyring' ) );
 			}
-
 		}
 
 		// Set up our defaults
 		$service = '';
-		if ( !empty( $_REQUEST['service'] ) )
+		if ( ! empty( $_REQUEST['service'] ) ) {
 			$service = $_REQUEST['service'];
+		}
 
 		$action = 'tokens';
-		if ( isset( $_REQUEST['action'] ) && in_array( $_REQUEST['action'], array( 'tokens', 'services', 'request', 'verify', 'manage' ) ) ) {
+		if ( isset( $_REQUEST['action'] ) && in_array( $_REQUEST['action'], array( 'tokens', 'services', 'request', 'verify', 'manage' ), true ) ) {
```

Comment:
Similar to above, this is a change to the logic, but I think $_REQUEST['action'] will always be a string so that's ok.